jquant1.c

```diff
@@ -531,6 +531,10 @@ quantize_ord_dither (j_decompress_ptr cinfo, JSAMPARRAY input_buf,
   JDIMENSION col;
   JDIMENSION width = cinfo->output_width;
 
+  if (output_buf == NULL && num_rows) {
+    ERREXIT(cinfo, JERR_BAD_PARAM);
```

Comment:
I do not see JERR_BAD_PARAM defined, am I missing something?

Reply:
Indeed, I forgot to include it in the patch. Fixed.

Reply:
Thanks!